perspective_reddit_bot/README.md

```diff
@@ -0,0 +1,44 @@
+# Perspective Moderation Bot
+
+## Description
+
+This repository contains a python script for a Reddit bot which uses the Perspective API to help moderate comments on a subreddit. By specifying rules in the rules.yaml file, the bot will check new comments on a given subreddit for triggering these rules and can report those that do. In order for the actions to be applied, the reddit credentials provided to the bot must have moderation permissions on the subreddit being moderated.
+
+## Setup
+
+1.  Setup a (virtualenv)[https://virtualenvwrapper.readthedocs.io/en/latest/] for
+    the project (recommended, but technically optional).
+
+    Python 2:
+
+    ```
+    python -m virtualenv env
```

Comment:
You are naming here your virtual environment `env`, but your `.ignore` rules specify `.env`; you want these to be consistent to help avoid checking in your venv files.